gwlfe.py

```diff
@@ -0,0 +1,694 @@
+AnnSumGroundWatLE = np.zeros((50, 12))
+AnnSumRunoff = np.zeros((50, 12))
+AnnSumStreamFlow = np.zeros((50, 12))
+AnnSumPtSrcFlow = np.zeros((50, 12))
+AnnSumTileDrain = np.zeros((50, 12))
+AnnSumWithdrawal = np.zeros((50, 12))
+AnnSumErosion = np.zeros((50, 12))
+AnnSumStreamBankEros = np.zeros((50, 12))
+AnnSumSedYield = np.zeros((50, 12))
+AnnSumGroundNitr = np.zeros((50, 12))
+AnnSumGroundPhos = np.zeros((50, 12))
+AnnSumDisNitr = np.zeros((50, 12))
+AnnSumTotNitr = np.zeros((50, 12))
+AnnSumDisPhos = np.zeros((50, 12))
+AnnSumTotPhos = np.zeros((50, 12))
+AnnSumStreamBankN = np.zeros((50, 12))
+AnnSumStreamBankP = np.zeros((50, 12))
+AnnSumTileDrainN = np.zeros((50, 12))
+AnnSumTileDrainP = np.zeros((50, 12))
+AnnSumTileDrainSed = np.zeros((50, 12))
+AnnSumAnimalN = np.zeros((50, 12))
+AnnSumAnimalP = np.zeros((50, 12))
+AnnSumAnimalFC = np.zeros((50, 12))
+AnnSumWWOrgs = np.zeros((50, 12))
+AnnSumSSOrgs = np.zeros((50, 12))
+AnnSumUrbOrgs = np.zeros((50, 12))
+AnnSumWildOrgs = np.zeros((50, 12))
+AnnSumTotalOrgs = np.zeros((50, 12))
+AnnSumCMStream = np.zeros((50, 12))
+AnnSumOrgConc = np.zeros((50, 12))
+AnnSumLuRunoff = np.zeros((50, 25))
+AnnSumLuErosion = np.zeros((50, 25))
+AnnSumLuSedYield = np.zeros((50, 25))
+AnnSumLuDisNitr = np.zeros((50, 25))
+AnnSumLuTotNitr = np.zeros((50, 25))
+AnnSumLuDisPhos = np.zeros((50, 25))
+AnnSumLuTotPhos = np.zeros((50, 25))
+AnnSumSepticNitr = np.zeros(50)
+AnnSumSepticPhos = np.zeros(50)
+AnnSumTotAnimalN = np.zeros(50)
+AnnSumTotAnimalP = np.zeros(50)
+AnnSumTotTileDrainSed = np.zeros(50)
+AnnSumTotTileDrainN = np.zeros(50)
+AnnSumTotTileDrainP = np.zeros(50)
+AnnSumTotGroundNitr = np.zeros(50)
+AnnSumTotGroundPhos = np.zeros(50)
+AnnSumAnnPointNitr = np.zeros(50)
+AnnSumAnnPointPhos = np.zeros(50)
+AnnSumTotStreamBankEros = np.zeros(50)
+AnnSumTotStreamBankN = np.zeros(50)
+AnnSumTotStreamBankP = np.zeros(50)
+
+# TODO: Add code to validate input, set up output files
+# Lines 1078 to 1140
+
+# ########################- BEGIN MAIN GWLF RUN CODE -########################
+# #>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>><<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<#
+# #>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>><<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<#
+# ############################################################################
+
+# CALCLULATE PRELIMINARY INITIALIZATIONS AND VALUES FOR
+# WATER BALANCE AND NUTRIENTS
+# TODO: Port this module
+# Call PrelimCalculations.InitialCalculations
+
+# MODEL CALCULATIONS FOR EACH YEAR OF ANALYSIS - WATER BALANCE,
+# NUTRIENTS AND SEDIMENT LOADS
+
+# TODO: Placeholder declaration
+NYrs = 10
+
+for y in range(1, NYrs):
+    # TODO: Placeholder declarations.
+    MonthPondNitr = np.zeros(12)
+    MonthPondPhos = np.zeros(12)
+    MonthNormNitr = np.zeros(12)
+    MonthShortNitr = np.zeros(12)
+    MonthShortPhos = np.zeros(12)
+    MonthDischargeNitr = np.zeros(12)
+    MonthDischargePhos = np.zeros(12)
+
+    # FOR EACH MONTH...
+    for i in range(1, 12):
```

Comment:
The end point in `range` is exclusive, so this iterates from 1 to 11. Because the above variables have a length of 12 and are 0-based, the first month is skipped.